main.tf

```diff
@@ -5,9 +5,11 @@ terraform {
 data "google_project" "project" {}
 
 locals {
-  google_pubsub_sa_email = "service-${data.google_project.project.number}@gcp-sa-pubsub.iam.gserviceaccount.com"
-  default_expiry_ttl     = var.default_expiry_ttl == "NEVER" ? "" : var.default_expiry_ttl
-  topic_name             = format("%s-%s", var.topic_name, var.name_suffix)
+  google_pubsub_sa_email  = "service-${data.google_project.project.number}@gcp-sa-pubsub.iam.gserviceaccount.com"
+  default_expiry_ttl      = var.default_expiry_ttl == "NEVER" ? "" : var.default_expiry_ttl
+  topic_name              = format("%s-%s", var.topic_name, var.name_suffix)
+  default_minimum_backoff = var.default_minimum_backoff == "10s" ? "" : var.default_minimum_backoff
+  default_maximum_backoff = var.default_maximum_backoff == "600s" ? "" : var.default_maximum_backoff
```

Comment:
What is this logic for?

Reply:
We needed this logic for `default_expiry_ttl` because terraform resources do not recognize what `"NEVER"` means. -so we had to sanitize that value from `"NEVER"` to `""`.

For `default_min/max_backoff` we don't have to worry about that as the value we are passing is itself recognized by terraform correctly

Reply:
I thought same default value need to be update in this section. 

I removed that condition now because those default value will be picked automatically from variables section.